projects/govuk-chat/Makefile

```diff
@@ -0,0 +1,4 @@
+govuk-chat: bundle-govuk-chat
+	$(GOVUK_DOCKER) run $@-lite bin/rails db:setup
+	$(GOVUK_DOCKER) run $@-lite env RAILS_ENV=test bin/rails db:setup
```

Comment:
`db:prepare` is a better option than `db:setup` as per: https://github.com/alphagov/govuk-docker/commit/b6eb8eb517fd18e9d97a0b7e6d4e0f2eef3f47e1